bzl/tools/gitget.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/sh
```

Comment:
Can you document this script, or remove it?

As it stands, it also seems to be equivalent to
```bash
git checkout --no-recurse-submodules $MINA_BRANCH
git submodule foreach --recursive git checkout $BRANCH
# straighten out other modules
```